app/webhook_test.go

```diff
@@ -456,6 +456,86 @@ func TestSplitWebhookPost(t *testing.T) {
 	}
 }
 
+func makePost(message int, attachments []int) *model.Post {
+	var props model.StringInterface
+	if len(attachments) > 0 {
+		sa := make([]*model.SlackAttachment, 0, len(attachments))
+		for _, a := range attachments {
+			attach := &model.SlackAttachment{
+				Text: strings.Repeat("那", a),
+			}
+			sa = append(sa, attach)
+		}
+		props = map[string]interface{}{"attachments": sa}
+	}
+	post := &model.Post{
+		Message: strings.Repeat("那", message),
+		Props:   props,
+	}
+	return post
+}
+
+func TestSplitWebhookPostAttachments(t *testing.T) {
+	maxPostSize := 10000
+	testCases := []struct {
+		name     string
+		post     *model.Post
+		expected []*model.Post
+	}{
+		{
+			// makePost(messageLength, []int{attachmentLength, ...})
```

Comment:
Let's remove this ?

Reply:
I left it there intentionally as documentation for the next person who is wondering what the numbers represent.  I saves them looking for the makePost func.

Reply:
Thanks for the review.  Any thoughts on whether the algorithm should be corrected now, so that the attachments are included starting with the first split?

Reply:
Ah I see. Yes that makes sense. Re: algorithm, I think it may be good to discuss this with the team. I am not sure on the historical context on the splitting decision here.